tests/stop-controllers.py

```diff
@@ -1,10 +1,13 @@
 #!/usr/bin/env python
+from __future__ import print_function
 import rospy
 from controller_manager_msgs.srv import SwitchController
 
 rospy.init_node('StoppingControllers')
 swCtl = rospy.ServiceProxy('/controller_manager/switch_controller', SwitchController)
-resp = swCtl(stop_controllers=[
-    'mobile_base_controller', 'arm_controller', 'head_controller', 'torso_controller', 'gripper_controller'
-],
-             strictness=2)
+for ctrl in [ 'mobile_base_controller', 'arm_controller', 'head_controller', 'torso_controller', 'hand_controller', 'gripper_controller' ]:
```

Comment:
I didn't check this script. Is the name of the controller `hand_controller` correct ?

Reply:
yes, it's hand_controller